packages/internal-scripts/src/add-browser-exports.ts

```diff
@@ -0,0 +1,272 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// This script exists to perform a one time operation of adding the "browser"
+// exports field to relevant package.json files. Motivated by:
+// https://github.com/lit/lit/issues/4446
+//
+// The script is idempotent.
+//
+// # Instructions
+// After building the script run:
+// `node packages/internal-scripts/bin/add-browser-exports.js`
+
+import assert from 'assert';
+import {execSync} from 'child_process';
+import {readFile, writeFile} from 'fs/promises';
+import {resolve, relative} from 'path';
+
+testScriptIntegrity();
+
+/**
+ * A loose structure for what we might find in the "exports" value within a
+ * `package.json` file.
+ */
+interface Exports {
+  [path: string]: {
+    types?: string;
+    browser?: {
+      development?: string;
+      default?: string;
+    };
+    node?: {
+      development?: string;
+      default?: string;
+    };
+    development?: string;
+    default?: string;
+  };
+}
+
+const result = execSync(`find . -name package.json`);
```

Comment:
would this maybe save at least grabbing `package.json` from root node_modules directory?
```suggestion
const result = execSync(`find ./packages -name package.json`);
```

Reply:
although keeping it `.` means you can run this from any package sub directory if you care to just test that one package. 🤔 feel free to ignore this one.

Reply:
yes! Great suggestion!